book/integration-service.md

```diff
@@ -0,0 +1,271 @@
+# Integration Service
+
+
+## Overview
+
+The Integration Service is composed of a couple controllers that facilitate automated testing of content produced by the HACBS build pipelines. It is responsible for integration testing capabilities.
+
+To support this capability, it needs to do a couple of things.
+
+- When a build pipeline completes, it creates an Snapshot CR representing a new collection of components that should be tested together.
+- When it sees a new Snapshot CR (created either by itself or by a user), it coordinates deployment of the application for testing. It does this by creating a new “ephemeral” StoneSoup Environment CRs which should in turn trigger the GitOps Service to provision environment (and a sub-workspace in CPS)
+- When it sees the environment is ready, it triggers Tekton PipelineRuns supplied by the user that run application tests and validation against the app running in that ephemeral Environment.
```

Comment:
Change the beginning to `After the environment is ready`. 

Does this wording inaccurately represent the service or fail to clarify its role? 
```
it deploys, tests, and validates the application in that ephemeral Environment according to user-provided configuration```

Reply:
Ah, I might not be accurate according to the words in the ephemeral environment. Integration service doesn't do a deployment, but the GitOps service deploys a set of application components as defined by a snapshot.

Reply:
Yes, I think that this point is accurate in its current state, we can change the beginning to be `After the environment is ready` and leave the rest as-is.

Reply:
Trying again to improve clarity here. How about something like this?
> After the environment is ready and the application snapshot has been deployed to it, the integration service tests and validates the application according to user-provided configuration.